### DISCUSSION TITLE
My cuda version is 11.4, so I can't use the torch, can I downgrade the version of torch, torchvision, torch-lightning and so on

### DISCUSSION THREAD

**Discussion by greatvalley** [open-edge-platform/anomalib#1291] [Q&A]
### What is the motivation for this task?

![Uploading image.png…]()

### Describe the solution you'd like

I want to use the anomalib to do some industry task, but I don't know whether the dependencies between packages can be change, please help me, thank you

### Additional context

![image](https://github.com/openvinotoolkit/anomalib/assets/102288830/fba342b1-ecb8-4d19-8b3a-3005b749d92f)

**Comment by greatvalley** [comment]
![image](https://github.com/openvinotoolkit/anomalib/assets/102288830/beed5b36-5de5-4a41-8afc-46f76f3af1a0)

**Comment by samet-akcay** [comment]
@greatvalley, have you tried using another torch version? If you install torch via pip or conda, it already comes with pre-built cuda, so your version should not matter unless you have a really old gpu card.

**Comment by samet-akcay** [comment]
I'm converting this to a Q&A in discussions. Let's continue there..
